playbook/app/pb_kits/playbook/pb_date_picker/date_picker_helper.ts

```diff
@@ -54,7 +73,7 @@ const datePickerHelper = (config, scrollContainer) => {
     }
   }
   const calendarResizer = () => {
-    const cal = document.querySelector(`#cal-${pickerId}.open`)
+    const cal = document.querySelector(`#cal-${pickerId}.open`) as HTMLElement | null
```

Comment:
```suggestion
    const cal = document.querySelector(`#cal-${pickerId}.open`) as HTMLElement
```

I would almost want to remove `null` more here then..